Task 8/script.js

```diff
@@ -5,3 +5,13 @@ subtraction(a, b) - priima du skaičius ir grąžina jų skirtumą.
 multiplication(a, b) - priima du skaičius ir grąžina jų daugybos rezultatą;
 division(a, b) - priima du skaičius ir grąžina jų dalybos rezultatą;
 ------------------------------------------------------------------------------------ */
+function Calculator() {}
+
+Calculator.prototype.sum = (a, b) => a + b;
```

Comment:
ir čia jau nebepaduot parametrų: 
Calculator.prototype.sum = function () {
   return this.a + this.b
}